vm-testing/test/test-sign-blob.sh

```diff
@@ -13,31 +13,45 @@ cp "${BASE_HOSTNAME}".pem /etc/pki/ca-trust/source/anchors/
 update-ca-trust
 
 # set up cosign env
-export KEYCLOAK_REALM=trusted-artifact-signer
+export OIDC_ISSUER_URL=http://dex-idp:5556/dex
```

Comment:
Let's make the OIDC_ISSUER_URL come from outside of the script to make it a little more flexible.